ip-location-service.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const path = require("path");
+const reader = require("@maxmind/geoip2-node").Reader;
+const AppConstants = require("./app-constants");
+
+let locationDb, timestamp;
```

Comment:
`timestamp` seems like a risky variable name for something that's being declared in global scope. Change to `locationDBLastOpenedTimestamp` or something?

Reply:
@pdehaan per my chat with @groovecoder:
`timestamp` is indeed scoped to that module:
`let locationDb, timestamp;` is short for
```
let locationDb;
let timestamp;
```
and both vars exist within a Node module which is scoped to that file alone.

Reply:
Yeah, I was thinking of https://eslint.org/docs/rules/one-var. But looking through the eslint source, I don't think that `one-var` is in the default `eslint:recommended` config. Carry on.